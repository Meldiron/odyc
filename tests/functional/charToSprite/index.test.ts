import { expect, test } from 'vitest'
import { charToSprite } from '../../../dist'

test('add to cell', async () => {
	'0123456789abcdefghijklmnopqrstuvwsyzABCDEFGHIJKLMNOPQRSTUVWSYZ'
		.split('')
		.forEach((c) => {
			expect(charToSprite(c, '1')).toMatchSnapshot()
		})
})
