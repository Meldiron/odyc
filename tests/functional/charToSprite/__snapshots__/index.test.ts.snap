// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`add to cell 1`] = `
".11111..
11...11.
11..111.
11.1111.
1111.11.
111..11.
.11111..
........
"
`;

exports[`add to cell 2`] = `
"..11....
.111....
..11....
..11....
..11....
..11....
111111..
........
"
`;

exports[`add to cell 3`] = `
".1111...
11..11..
....11..
..111...
.11.....
11..11..
111111..
........
"
`;

exports[`add to cell 4`] = `
".1111...
11..11..
....11..
..111...
....11..
11..11..
.1111...
........
"
`;

exports[`add to cell 5`] = `
"...111..
..1111..
.11.11..
11..11..
1111111.
....11..
...1111.
........
"
`;

exports[`add to cell 6`] = `
"111111..
11......
11111...
....11..
....11..
11..11..
.1111...
........
"
`;

exports[`add to cell 7`] = `
"..111...
.11.....
11......
11111...
11..11..
11..11..
.1111...
........
"
`;

exports[`add to cell 8`] = `
"111111..
11..11..
....11..
...11...
..11....
..11....
..11....
........
"
`;

exports[`add to cell 9`] = `
".1111...
11..11..
11..11..
.1111...
11..11..
11..11..
.1111...
........
"
`;

exports[`add to cell 10`] = `
".1111...
11..11..
11..11..
.11111..
....11..
...11...
.111....
........
"
`;

exports[`add to cell 11`] = `
"........
........
.1111...
....11..
.11111..
11..11..
.111.11.
........
"
`;

exports[`add to cell 12`] = `
"111.....
.11.....
.11.....
.11111..
.11..11.
.11..11.
11.111..
........
"
`;

exports[`add to cell 13`] = `
"........
........
.1111...
11..11..
11......
11..11..
.1111...
........
"
`;

exports[`add to cell 14`] = `
"...111..
....11..
....11..
.11111..
11..11..
11..11..
.111.11.
........
"
`;

exports[`add to cell 15`] = `
"........
........
.1111...
11..11..
111111..
11......
.1111...
........
"
`;

exports[`add to cell 16`] = `
"..111...
.11.11..
.11.....
1111....
.11.....
.11.....
1111....
........
"
`;

exports[`add to cell 17`] = `
"........
........
.111.11.
11..11..
11..11..
.11111..
....11..
11111...
"
`;

exports[`add to cell 18`] = `
"111.....
.11.....
.11.11..
.111.11.
.11..11.
.11..11.
111..11.
........
"
`;

exports[`add to cell 19`] = `
"..11....
........
.111....
..11....
..11....
..11....
.1111...
........
"
`;

exports[`add to cell 20`] = `
"....11..
........
....11..
....11..
....11..
11..11..
11..11..
.1111...
"
`;

exports[`add to cell 21`] = `
"111.....
.11.....
.11..11.
.11.11..
.1111...
.11.11..
111..11.
........
"
`;

exports[`add to cell 22`] = `
".111....
..11....
..11....
..11....
..11....
..11....
.1111...
........
"
`;

exports[`add to cell 23`] = `
"........
........
11..11..
1111111.
1111111.
11.1.11.
11...11.
........
"
`;

exports[`add to cell 24`] = `
"........
........
11111...
11..11..
11..11..
11..11..
11..11..
........
"
`;

exports[`add to cell 25`] = `
"........
........
.1111...
11..11..
11..11..
11..11..
.1111...
........
"
`;

exports[`add to cell 26`] = `
"........
........
11.111..
.11..11.
.11..11.
.11111..
.11.....
1111....
"
`;

exports[`add to cell 27`] = `
"........
........
.111.11.
11..11..
11..11..
.11111..
....11..
...1111.
"
`;

exports[`add to cell 28`] = `
"........
........
11.111..
.111.11.
.11..11.
.11.....
1111....
........
"
`;

exports[`add to cell 29`] = `
"........
........
.11111..
11......
.1111...
....11..
11111...
........
"
`;

exports[`add to cell 30`] = `
"...1....
..11....
.11111..
..11....
..11....
..11.1..
...11...
........
"
`;

exports[`add to cell 31`] = `
"........
........
11..11..
11..11..
11..11..
11..11..
.111.11.
........
"
`;

exports[`add to cell 32`] = `
"........
........
11..11..
11..11..
11..11..
.1111...
..11....
........
"
`;

exports[`add to cell 33`] = `
"........
........
11...11.
11.1.11.
1111111.
1111111.
.11.11..
........
"
`;

exports[`add to cell 34`] = `
"........
........
.11111..
11......
.1111...
....11..
11111...
........
"
`;

exports[`add to cell 35`] = `
"........
........
11..11..
11..11..
11..11..
.11111..
....11..
11111...
"
`;

exports[`add to cell 36`] = `
"........
........
111111..
1..11...
..11....
.11..1..
111111..
........
"
`;

exports[`add to cell 37`] = `
"..11....
.1111...
11..11..
11..11..
111111..
11..11..
11..11..
........
"
`;

exports[`add to cell 38`] = `
"111111..
.11..11.
.11..11.
.11111..
.11..11.
.11..11.
111111..
........
"
`;

exports[`add to cell 39`] = `
"..1111..
.11..11.
11......
11......
11......
.11..11.
..1111..
........
"
`;

exports[`add to cell 40`] = `
"11111...
.11.11..
.11..11.
.11..11.
.11..11.
.11.11..
11111...
........
"
`;

exports[`add to cell 41`] = `
"1111111.
.11...1.
.11.1...
.1111...
.11.1...
.11...1.
1111111.
........
"
`;

exports[`add to cell 42`] = `
"1111111.
.11...1.
.11.1...
.1111...
.11.1...
.11.....
1111....
........
"
`;

exports[`add to cell 43`] = `
"..1111..
.11..11.
11......
11......
11..111.
.11..11.
..11111.
........
"
`;

exports[`add to cell 44`] = `
"11..11..
11..11..
11..11..
111111..
11..11..
11..11..
11..11..
........
"
`;

exports[`add to cell 45`] = `
".1111...
..11....
..11....
..11....
..11....
..11....
.1111...
........
"
`;

exports[`add to cell 46`] = `
"...1111.
....11..
....11..
....11..
11..11..
11..11..
.1111...
........
"
`;

exports[`add to cell 47`] = `
"111..11.
.11..11.
.11.11..
.1111...
.11.11..
.11..11.
111..11.
........
"
`;

exports[`add to cell 48`] = `
"1111....
.11.....
.11.....
.11.....
.11...1.
.11..11.
1111111.
........
"
`;

exports[`add to cell 49`] = `
"11...11.
111.111.
1111111.
1111111.
11.1.11.
11...11.
11...11.
........
"
`;

exports[`add to cell 50`] = `
"11...11.
111..11.
1111.11.
11.1111.
11..111.
11...11.
11...11.
........
"
`;

exports[`add to cell 51`] = `
"..111...
.11.11..
11...11.
11...11.
11...11.
.11.11..
..111...
........
"
`;

exports[`add to cell 52`] = `
"111111..
.11..11.
.11..11.
.11111..
.11.....
.11.....
1111....
........
"
`;

exports[`add to cell 53`] = `
".1111...
11..11..
11..11..
11..11..
11.111..
.1111...
...111..
........
"
`;

exports[`add to cell 54`] = `
"111111..
.11..11.
.11..11.
.11111..
.11.11..
.11..11.
111..11.
........
"
`;

exports[`add to cell 55`] = `
".1111...
11..11..
111.....
.111....
...111..
11..11..
.1111...
........
"
`;

exports[`add to cell 56`] = `
"111111..
1.11.1..
..11....
..11....
..11....
..11....
.1111...
........
"
`;

exports[`add to cell 57`] = `
"11..11..
11..11..
11..11..
11..11..
11..11..
11..11..
111111..
........
"
`;

exports[`add to cell 58`] = `
"11..11..
11..11..
11..11..
11..11..
11..11..
.1111...
..11....
........
"
`;

exports[`add to cell 59`] = `
"11...11.
11...11.
11...11.
11.1.11.
1111111.
111.111.
11...11.
........
"
`;

exports[`add to cell 60`] = `
".1111...
11..11..
111.....
.111....
...111..
11..11..
.1111...
........
"
`;

exports[`add to cell 61`] = `
"11..11..
11..11..
11..11..
.1111...
..11....
..11....
.1111...
........
"
`;

exports[`add to cell 62`] = `
"1111111.
11...11.
1...11..
...11...
..11..1.
.11..11.
1111111.
........
"
`;
